lua/diffview/init.lua

```diff
@@ -23,6 +23,22 @@ end
 
 function M.init()
   local au = api.nvim_create_autocmd
+
+  -- Fix the strange behavior that "<afile>" expands non-files
+  -- as file name in some cases.
+  --
+  -- About more information please checkout:
+  --  * sindrets/diffview.nvim#369
+  --  * neovim/neovim#23943
+  local fetch_tabnr = function (state)
```

Comment:
Let's just call it `get_tabnr`.

Nit pick: I prefer using Lua's syntactic sugar for defining functions. It's a little denser and it makes a clearer distinction between functions and other types.

```suggestion
  local function get_tabnr(state)
```

Reply:
Great shout! This does get better and has been fixed.